python/hail/expr/aggregators.py

```diff
@@ -119,6 +517,45 @@ def fraction(expr):
 
 
 def hardy_weinberg(expr):
+    """Compute Hardy-Weinberg Equilbrium (HWE) p-value and heterozygosity ratio.
+
+    Examples
+    --------
+    Compute HWE statistics per row of a dataset:
+
+    .. doctest::
+
+        >>> dataset_result = dataset.annotate_rows(hwe = agg.hardy_weinberg(dataset.GT))
+
+    Compute HWE statistics for a single population:
+
+    .. doctest::
+
+        >>> dataset_result = dataset.annotate_rows(
+        ...     hwe_eas = agg.hardy_weinberg(agg.filter(dataset.GT, dataset.pop == 'EAS')))
+
+    Notes
+    -----
+    This method returns a struct expression with the following fields:
+     - `rExpectedHetFrequency` (:class:`hail.expr.types.TFloat64`) - Ratio of observed to
```

Comment:
Table malformed... Extra newline?

Reply:
why did you `approve` this dumpster fire? Will fix.

Reply:
Well, "approved" pending these tiny little changes. I guess I should have rejected, just felt harsh.